_data/identity.yml

```diff
@@ -1,4 +1,15 @@
 websites:
+    - name: 1Password
+      url: https://1password.com/
+      img: 1password.png
+      tfa: No
+      email_address: support@1password.com
+      facebook: 1Password
+      twitter: 1Password
+      exceptions:
+        text: "AgileBits, the company behind 1Password, feels that two factor authentication doesn't really fit into 1Password's offering. We disagree because of the way in which 1Password handles authentication and password data."
```

Comment:
I feel this is an unnecessary qualification. Ultimately the reasons behind why 2fa is not implemented are irrelevant. Especially now that 1pass functions similarly to other online password managers. Your password database is no longer kept offline-only.